polaris-core/src/main/java/io/polaris/core/persistence/LocalPolarisMetaStoreManagerFactory.java

```diff
@@ -102,12 +110,18 @@ public void purgeRealms(List<String> realms) {
       backingStoreMap.remove(realm);
       sessionSupplierMap.remove(realm);
       metaStoreManagerMap.remove(realm);
+      bootstrappedRealms.remove(realm);
     }
   }
 
   @Override
   public synchronized PolarisMetaStoreManager getOrCreateMetaStoreManager(
       RealmContext realmContext) {
+    String realmId = realmContext.getRealmIdentifier();
+    if (bootstrapEnabled && !bootstrappedRealms.contains(realmId)) {
+      bootstrapRealmAndPrintCredentials(realmId);
+    }
+
```

Comment:
This code was intentionally moved out of here. Nobody wants to accidentally bootstrap their production instance. Including this code here allows for a misconfigured server instance to irrevocably wipe out all production data. Instead, let's just update the tests to explicitly call bootstrap on the metastore manager

Reply:
That makes sense.